src/pages/workspace/companyCards/WorkspaceCompanyCardsFeedNoFeedPendingPage.tsx

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+import EmptyStateComponent from '@components/EmptyStateComponent';
+import * as Illustrations from '@components/Icon/Illustrations';
+import TableListItemSkeleton from '@components/Skeletons/TableRowSkeleton';
+import TextLink from '@components/TextLink';
+import useLocalize from '@hooks/useLocalize';
+import useThemeStyles from '@hooks/useThemeStyles';
+import colors from '@styles/theme/colors';
+import * as ReportInstance from '@userActions/Report';
+import CONST from '@src/CONST';
+
+function WorkspaceCompanyCardsFeedNoFeedPendingPage() {
```

Comment:
```suggestion
function WorkspaceCompanyCardsFeedPendingPage() {
```
Any reason to have it FeedNoFeed?